extensions/msal-node-extensions/rollup.config.js

```diff
@@ -57,7 +59,8 @@ export default [
         },
         external: [
             ...Object.keys(pkg.dependencies || {}),
-            ...Object.keys(pkg.peerDependencies || {})
+            ...Object.keys(pkg.peerDependencies || {}),
+            ...Object.keys(pkg.bundledDependencies || {}),
```

Comment:
This continues to treat `bundledDependencies` as external (i.e. not bundled). What we want is for rollup to actually resolve the `bundledDependencies` to they get included in the build output